### PR TITLE
✨ ENHANCE: expand Dable AMP options

### PR DESCRIPTION
**Pull request by Minjiicho**
- Add new option for special case
- Add backup options to deliver accurate data

### CODE REVIEW

**Comment by Minjiicho** [comment]
Hi @calebcordry 
Could you please advise why workflow jobs are not progressing and are on standby? 🙏

**Review by erwinmombay — APPROVED** [review]
LGTM but will leave final approval to @powerivq

File: ads/vendors/dable.js [hunk 1]
```diff
@@ -52,6 +56,10 @@ export function dable(global, data) {
     widgetOpts.category3 = articleSection3;
     logOpts.category3 = articleSection3;
   }
+  if (orgServiceId) {
+    widgetOpts.orgServiceId = orgServiceId;
```
Comment (erwinmombay) [inline ads/vendors/dable.js:60]: @powerivq these don't get obfuscated?

Comment (powerivq) [reply ads/vendors/dable.js:60]: I think it's up to Dable whether to do something about these parameters.

**Review by powerivq — APPROVED** [review]

File: ads/vendors/dable.js (hunk 1 above)
Comment (Minjiicho) [reply ads/vendors/dable.js:60]: @erwinmombay Thank you for the review 🙏 
It's an internally agreed value, so I don't think it'll be a problem but please let me know if you have any concerns.

**Comment by powerivq** [comment]
For some reasons, circle build is not running. Can you rebase so that it can retry? @Minjiicho

**Comment by Minjiicho** [comment]
> For some reasons, circle build is not running. Can you rebase so that it can retry? @Minjiicho

@powerivq I made trivial update to the guide. Hope CircleCI rerun 🙏

**Comment by Minjiicho** [comment]
@powerivq CircleCI is still on waiting 🤔 
Please let me know if I can try someting.

**Comment by powerivq** [comment]
@Minjiicho I suspect it is because the branch name contains `/`. You can try recreating a PR and see if it works.

**Comment by Minjiicho** [comment]
> @Minjiicho I suspect it is because the branch name contains `/`. You can try recreating a PR and see if it works.

@powerivq  Thx for sharing suspicious part. Let me close this PR since reopened new PR #39433

**Pull request closed without merging**